challenge-03/TomcatWebServer.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: ubuntu
+  tasks:
+    - name: Install Tomcat 9 on Ubuntu
+      become: yes
+      apt: pkg={{ item }} state=latest update_cache=yes cache_valid_time=3600
+      with_items:
```

Comment:
1. Use _loop_ instead of _with_items_. Ansible documentation recommend this.
2. I would avoid using loop since the list only contains 1 item and it is slower.